R/sparql.R

```diff
@@ -0,0 +1,176 @@
+#' @title SPARQLmap
+#' @name SPARQLmap
+#' @param x `Character vector`. Optional. Feature IDs on which the query should
+#'     be constrained.
+#' @param x.type `Character scalar`. Feature type of `x`. (Default: `"uniref"`)
+#' @param .by either a `formula`, or a `Character vector` of length 2,  with the
+#'      names of the desired combination of feature types.
+#' @param endpoint The SPARQL endpoint (a URL)
+#' @returns a data.frame of desired query.
+#' @export
+#' @examples
+#' # x <- levels(
+#' #     ariadne::importMapping(
+#' #         ariadne::ariadne("choco", uniref90 ~ ko),
+#' #         dry.run = FALSE
+#' #         )
+#' #     )[["uniref90"]][1:100]
+#'
+#'
+#' x <-
```

Comment:
For this demo, a smaller vector of uniref can be used to avoid too long vignettes.